test/test.js

```diff
@@ -44,8 +44,8 @@ describe('eth-net-props', () => {
 			it(`${claimPrefix} Ethereum Classic`, () => {
 				assert.equal(explorerLinks.getExplorerAccountLinkFor('0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0', 61), 'https://blockscout.com/etc/mainnet/address/0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0')
 			})
-			it(`${claimPrefix} Ethereum Classic`, () => {
-				assert.equal(explorerLinks.getExplorerAccountLinkFor('0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0', '61'), 'https://blockscout.com/etc/mainnet/address/0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0')
+			it(`${claimPrefix} Callisto Network`, () => {
+				assert.equal(explorerLinks.getExplorerAccountLinkFor('0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0', 820), 'https://explorer2.callisto.network/address/0xd8fe15886d2dcbc5d7c06394beb417aadaf1eee0')
```

Comment:
I've checked that address page in Callisto explorer is available at `/addr/...` path and not at `/address/`. I suppose path should be changed here in the tests and in the logic.